src/frontend/components/LTINav/index.spec.tsx

```diff
@@ -0,0 +1,167 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+import { wrapInIntlProvider } from '../../utils/tests/intl';
+import React from 'react';
+import { LTINav } from './index';
+import {
+  documentMockFactory,
+  videoMockFactory,
+} from '../../utils/tests/factories';
+import { uploadState, Video } from '../../types/tracks';
+import { Document } from '../../types/file';
+import { Router } from 'react-router-dom';
+import { PLAYER_ROUTE } from '../routes';
+import { DASHBOARD_ROUTE } from '../Dashboard/route';
+import { modelName } from '../../types/models';
+import { createMemoryHistory } from 'history';
+
+let mockModelName: any;
+let mockCanUpdate: boolean;
+let mockMaintenance: boolean;
+jest.mock('../../data/appData', () => ({
+  appData: {
+    get modelName() {
+      return mockModelName;
+    },
+  },
+  getDecodedJwt: () => ({
+    permissions: {
+      can_update: mockCanUpdate,
+    },
+    maintenance: mockMaintenance,
+  }),
+}));
+
+describe('<LTINav />', () => {
+  const history = createMemoryHistory();
+  const testLTINav = (object: Video | Document) =>
+    render(
+      wrapInIntlProvider(
+        <Router history={history}>
+          <LTINav object={object} />
+        </Router>,
+      ),
+    );
```

Comment:
IMO as it's usually done in the rest of the project, you can insert the render in each test, no need for a function.